src/test/java/de/tum/in/www1/artemis/exercise/textexercise/TextExerciseUtilService.java

```diff
@@ -346,12 +481,13 @@ public TextExercise createTextExerciseForExam(ExerciseGroup exerciseGroup) {
     }
 
     /**
-     * Creates a submission with a result and an assessor of the passed text exercise.
+     * Creates and saves a TextSubmission and StudentParticipation for the given TextExercise, and studentLogin. Also creates and saves a Result for the
```

Comment:
Could you mention the tutor Assessor here?

Reply:
could you also update it in the beginning? 
```suggestion
     * Creates and saves a TextSubmission and StudentParticipation for the given TextExercise, and studentLogin. Also creates and saves a Result for the StudentParticipation given the assessorLogin.
```

Reply:
Done, thanks for noticing :)